Backend/wrappers/standardAPI.py

```diff
@@ -54,6 +55,61 @@ def getTweets(self, keyword, since, until, num, location=None):
 
             return tweetsparsed
 
+    # Get twitter user profile with screen_name (e.g. @ttclaire2)
+    #   return type: a twitter User object
+    def get_user(self, public_id):
+        try:
+            user_object = self.tweepy.get_user(id=None, user_id=None, screen_name=public_id)
+            return user_object
+        except:
+            return "EXCEPTION: Twitter user cannot be found"
+
+    # Get the followers of the specified user 
+    #   return type: a list of twitter User objects
+    def get_followers(self, public_id, max_num):
+        followerList = []
+        try:
+            for follower in tw.Cursor(self.tweepy.followers, screen_name=public_id).items(max_num):
+                followerList.append(follower)
+            return followerList
+        except:
+            return "EXCEPTION: Page does not exist"
+
+    # Get the most recent tweets of the specified user
+    #   return: count passed as parameter, or the total number of available tweets
+    def get_user_timeline(self, public_id, max_num=30):
+        timeline = []
+        try:
+            for status in tw.Cursor(self.tweepy.user_timeline, screen_name=public_id).items(max_num):
+                timeline.append(status)
+            return timeline
+        
+        except:
+            return "EXCEPTION: Page does not exist"
+
+    # Get a list of random retweets for a given tweet ID
+    #   args: tweet ID, count (max = 100)
+    #   return type: a list of Twitter status objects 
+    # Sometimes the number returned is less than input
+    def get_retweets_of_tweet(self, tweet_id, max_num=100):
+        # retweetList = []
+        try:
+            retweets = self.tweepy.retweets(id=tweet_id, count=max_num)
+            return retweets
+        
+        except:
+            return "EXCEPTION: Invalid Tweet ID"
```

Comment:
Isn't this error message making the assumption that the problem was in the input and not somewhere in the server's logic or internal state?

Reply:
Good catch! Actually it seems like this is why I had said that the server crashed. It was probably because RateLimitError occurred instead of "Invalid Tweet ID", given that each time the server retrieves retweeters, it passes dozens of tweets and calls the API a lot.